src/index.ts

```diff
@@ -5,12 +5,11 @@ export * from './types';
 export * from './components';
 
 export {UnstyledLinkProps, LinkLikeComponent} from './utilities/link';
-export {createPolarisContext} from './utilities/create-polaris-context';
```

Comment:
Just as a quick note in a follow-up PR I can probably remove this entirely from polaris